go/vt/vtgate/vindexes/lookup_internal.go

```diff
@@ -13,18 +16,18 @@ type lookup struct {
 	isHashedIndex      bool
 }
 
-func (lkp *lookup) Init(m map[string]string, isHashed bool) {
-	t := m["table"]
-	from := m["from"]
-	to := m["to"]
+func (lkp *lookup) Init(lookupQueryParams map[string]string, isHashed bool) {
```

Comment:
Many functions in this file are missing critical code coverage. It's probably an artifact of previous changes.